.github/workflows/multidim-interop.yml

```
@@ -9,14 +9,15 @@ name: libp2p multidimensional interop test
 
 jobs:
   run-multidim-interop:
-    runs-on: ubuntu-latest
+    runs-on: ['self-hosted', 'linux', 'x64', '4xlarge'] # https://github.com/pl-strflt/tf-aws-gh-runner/blob/main/runners.tf
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/run-interop-ping-test
         with:
           s3-cache-bucket: libp2p-by-tf-aws-bootstrap
           s3-access-key-id: ${{ vars.S3_AWS_ACCESS_KEY_ID }}
           s3-secret-access-key: ${{ secrets.S3_AWS_SECRET_ACCESS_KEY }}
+          worker-count: 16
```

If we want to stick to hosted in this repo, let's apply this suggestion. I'd still want to have this PR merged because it'd enable other repos to modify worker count and fully take advantage of self-hosted config.

```suggestion
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: ./.github/actions/run-interop-ping-test
        with:
          s3-cache-bucket: libp2p-by-tf-aws-bootstrap
          s3-access-key-id: ${{ vars.S3_AWS_ACCESS_KEY_ID }}
          s3-secret-access-key: ${{ secrets.S3_AWS_SECRET_ACCESS_KEY }}
```

---

I am in favor of that.

Although I'd assume that the traffic on this repo in terms of PRs is negligible compared to our repos. That being said, I don't really care about how long CI takes in here so I think it makes sense to stick to hosted runners in this repo.